File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>
#include <math.h>

int count_letters (string text);
int count_words (string text);
int count_sentences (string text);



int main(void)
{
    string text;
    float index;
    int letters = 0;
    int sentences = 0;
    int words = 0;






    text = get_string("Text: ");


    float L = (float) letters / (float) words *100;
    float S = (float) sentences / (float) words *100;

     index = round(0.0588 * L - 0.296 * S - 15.8);
}

int count_letters (string text)
{
    int counter = 0;
    int length = strlen(text);

    for (int i = 0; i < length; i++)
    {
        counter++
    }

// monke 




}

File: sentimental-readability/readability.py
from cs50 import get_string

def main():
    text = get_string("Text: ")
    letters = count_letters(text)
    sentences = count_sentences(text)
    words = count_words(text)

    L = letters / words * 100
    S = sentences / words * 100

    index = round(0.0588 * L - 0.296 * S - 15.8)

    if index > 16:
        print("Grade 16+")
    elif index < 1:
        print("Before Grade 1")
    else:
        print("Grade ", + int(round(index)))

def count_words(text):
    counter = 1
    length = len(text)
    